client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -319,6 +324,47 @@ const getVisiblePlugins = ( plugins, country, industry, productTypes ) => {
 	);
 };
 
+const transformRemoteExtensions = ( extensionData, localeValue ) => {
+	return extensionData.reduce( ( result, extension ) => {
+		const localeData =
+			extension.locales.find(
+				( locale ) => locale.locale === localeValue
+			) ||
+			extension.locales.find( ( locale ) => locale.locale === 'en_US' );
+		const transformedExtension = {
+			title: localeData.title,
+			slug: extension.key,
+			description: generatePluginDescriptionWithLink(
+				localeData.description,
+				extension.key
+			),
+			isVisible: () => true,
+		};
+		const sectionIndex = extension.section === 'primary' ? 0 : 1;
+		result[ sectionIndex ].plugins.push( transformedExtension );
+		return result;
+	}, initialInstallableExtensions );
+};
+
+const baseValues = { install_extensions: true };
+const createInitialValues = ( extensions, country, industry, productTypes ) => {
+	return extensions.reduce( ( acc, curr ) => {
+		const plugins = getVisiblePlugins(
```

Comment:
Sorry if this was addressed elsewhere, but can we remove this in favor of rulesets?

Reply:
`createInitialValues` is used for both local and remote data, which is why I added `isVisible: () => true` to each of the remote items. I can see why this isn't straight forward. I was trying to leave the local data in its original state, but maybe its best to modify it to something more similar to the remote version so I don't need to transform as much. I will look into this.